file_test

```diff
@@ -0,0 +1,80 @@
+package controllers
```

Comment:
shouldn't this file be in a test folder (and with a `.go` extension) ?

Reply:
It s apparently not possible, but we didn't really try to figure out another way to do it